### ISSUE TITLE
Training

### ISSUE THREAD

**Issue by Adamdad** [ThomasMerkh/covid-ct#1]
Could you release your training code?

**Comment by Adamdad** [comment]
And I see in your code that you test on valiation set. What is the performance on  test set? Thank you.

**Comment by ThomasMerkh** [comment]
Yes I will, once it is cleaned up and I figure out a Tensorflow v1 (not v2) detail that will make the code user friendly for others.
